### PR TITLE
Add a descriptive README

### PR DESCRIPTION
**Pull request by nicholasjhenry**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,85 @@
...
+  * [Elixir](https://elixir-lang.org/): a dynamic, functional language designed for building scalable
+    and maintainable applications, and
+  * [Phoenix Framework](http://www.phoenixframework.org/): a productive web framework that does not
+    compromise speed and maintainability.
+
+The website has three main purposes:
+
+  * Promote the Elixir community in Montreal
+  * Inform the community members about events and new content
+  * Provide an opportunity to to learn about Elixir and Phoenix by contributing to an
```
Comment (frahugo) [inline README.md]: Remove a `to`

File: README.md [hunk 2]
```diff
@@ -1,9 +1,85 @@
...
+  * Node
+  * PostgreSQL
+
+### Installing
+
+After you have cloned the repository:
+
+* Install dependencies with `mix deps.get`
+* Create and migrate your database with `mix ecto.create && mix ecto.migrate`
+* Install Node.js dependencies with `cd assets && npm install`
```
Comment (frahugo) [inline README.md]: I think you mean `cd apps/montreal_elixir_web/assets && npm install `

File: README.md [hunk 3]
```diff
@@ -1,9 +1,85 @@
...
+### Prerequisites
+
+We assume you have the following installed:
+
+  * Git
+  * Erlang
+  * Elixir
+  * Node
+  * PostgreSQL
+
```
Comment (frahugo) [inline README.md:42]: Should we suggest `asdf` and include a `.tool-versions` at the root?

File: README.md [hunk 4]
```diff
@@ -1,9 +1,85 @@
...
+### Installing
+
+After you have cloned the repository:
+
+* Install dependencies with `mix deps.get`
+* Create and migrate your database with `mix ecto.create && mix ecto.migrate`
```
Comment (frahugo) [inline README.md]: Would it be better to use mix this way: `mix do ecto.create, ecto.migrate`?

File: README.md [hunk 5]
```diff
@@ -1,9 +1,85 @@
...
+* Install Node.js dependencies with `cd assets && npm install`
+* Start Phoenix endpoint with `mix phx.server`
```
Comment (frahugo) [inline README.md]: `cd ../../.. && mix phx.server`

**Review by frahugo — DISMISSED** [review]
Great README!  Just a few things

File: README.md (hunk 3 above)
Comment (nicholasjhenry) [reply README.md:42]: We'll leave this for now, we'll let folks work in whatever environment they want.

File: README.md (hunk 4 above)
Comment (nicholasjhenry) [reply README.md]: Funny, I took that right out of there docs ;-)

**Pull request merged**